packages/cli/lib/prompts/createProjectPrompt.js

```diff
@@ -91,8 +98,9 @@ const createProjectPrompt = async (githubRef, promptOptions = {}) => {
           : i18n(`${i18nKey}.selectTemplate`);
       },
       when:
-        !promptOptions.template ||
-        !projectTemplates.find(t => t.name === promptOptions.template),
+        (templatesRequired && !promptOptions.template) ||
```

Comment:
Not a blocker, but this could also be written as
```js
!skipTemplatePrompt && (!promptOptions.template || !projectTemplates.find(t => t.name === promptOptions.template))`
```
to avoid using templatesRequired twice (also this assumes you make the variable name change)

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1052/commits/0930b7255c8ed88d0f7e21420974c9223016f4e4